go.mod

```diff
@@ -422,7 +422,7 @@ replace (
 // Uncomment for local development for testing with changes in the components-contrib repository.
 // Don't commit with this uncommented!
 //
-// replace github.com/dapr/components-contrib => ../components-contrib
-//
+replace github.com/dapr/components-contrib => github.com/DeepanshuA/components-contrib v1.6.0-rc.2.0.20230104154635-e891f15b6403
```

Comment:
Reviewers: I will be removing this after https://github.com/dapr/components-contrib/pull/2405 is merged. And, this PR has to be merged only after it is removed from here.

Reply:
Removed local fork reference, as https://github.com/dapr/components-contrib/pull/2405 is merged.